### ISSUE TITLE
Raising the Firebase error instead of returning error 500 in admin

### ISSUE THREAD

**Issue by p02diada** [xtrinch/fcm-django#272]
Hi, 

There is a problem from version 2.0.1 with this PR https://github.com/xtrinch/fcm-django/pull/241 in the django Admin. If you try to send test notification to a device that firebase does not accept it raise a 500 error.

![image](https://github.com/user-attachments/assets/3b654a9e-134a-4b51-92b2-293c33dbcadd)

**Comment by xtrinch** [comment]
Yes looks like the admin functions should now be wrapped in a try catch - good find
